packages/node/src/child_process.ts

```diff
@@ -34,9 +34,14 @@ function *supervise(child: ChildProcess, command: string, args: readonly string[
 }
 
 export function *spawn(command: string, args?: ReadonlyArray<string>, options?: SpawnOptions): Operation {
+  // Use the process.env.shell if it isn't set. This will likely be most useful for Windows
+  // as there are more terminals and the defaults are not amazing.
+  // windowsHide also hides the terminals from showing when a child process is init. Setting the boolean
+  // to true makes this operate in line with other OS, and how one might expect.
   let child = childProcess.spawn(command, args || [], Object.assign({}, options, {
-    shell: true,
+    shell: !options && !options.shell ? process.env.shell : options.shell,
```

Comment:
@jnicklas what was the original reason for setting the `shell: true` property?

@jbolda since we're using a modern TypeScript, you should be able to say `options?.shell || process.env.shell`

Reply:
I think it's a bit of a mistake actually? We do want `shell: true` in bigtest, where this was extracted from, but we probably shouldn't always set it here.